crux-llvm/src/Crux/LLVM/Log.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Crux.LLVM.Log
+  ( CruxLLVMLogMessage (..),
+    cruxLLVMLogMessageToSayWhat,
+  )
+where
+
+import Crux.Log (SayLevel (..), SayWhat (..), cruxLogTag)
+import Data.Text as Text (Text, pack, unwords)
+
+data CruxLLVMLogMessage
+  = ClangInvocation [Text]
+  | DebugBreakpoint Text
+  | DebugExecutable Text
```

Comment:
Similar to my comment below, maybe we should remove the `Debug...` prefix from these messages since that imposes a severity.  I realize we'd probably need to replace it with _something_ since just `Breakpoint` and `Executable` are too short and non-unique.

Reply:
Yeah, and I do feel like the names here tend to be very general and are best used qualified. For instance, `CruxLogMessage` has constructors like `Goals`, `Help`, `Version` that are bound to collide with other constants, and I think it's better in general to always qualify them to avoid any confusion.